### DISCUSSION TITLE
Side Navigation not closing

### DISCUSSION THREAD

**Discussion by jeffreyh0** [olton/Metro-UI-CSS#1785] [Q&A] [answered]
Hi all!

I have been trying to test out the side navigation feature in Metro UI and I was not able to close it programmatically
The code below successfully gets the nav object but the close function will not close the side navigation bar. Any ideas what am I missing? Thank you!

Ref: [https://metroui.org.ua/navview.html](url)
   
   `
        
        $( document ).ready(function() {
            var nav = Metro.getPlugin("#navview1", 'navview');
            console.log(nav);
            
            setTimeout(function(){
                console.log("close sidenav");
                nav.close();
            }, 1000);
        });
`

**Comment by olton** [comment, marked as answer]
```
$( document ).ready(function() {
    var nav = Metro.getPlugin("#navview1", 'navview');
        
    setTimeout(function(){
        nav.pullClick(null); 
    }, 1000);
});
```

**Reply by jeffreyh0** [reply]
Thanks a lot!
